controller/space.go

```diff
@@ -376,6 +400,10 @@ func ConvertSpaceToModel(appSpace app.Space) space.Space {
 		appSpace.Relationships.OwnedBy.Data != nil && appSpace.Relationships.OwnedBy.Data.ID != nil {
 		modelSpace.OwnerId = *appSpace.Relationships.OwnedBy.Data.ID
 	}
+	if appSpace.Relationships != nil && appSpace.Relationships.SpaceTemplate != nil &&
```

Comment:
I don't want into a seg-fault panic when assigning `modelSpace.SpaceTemplateID = *appSpace.Relationships.SpaceTemplate.Data.ID`. That's all. And `validateCreateSpace` is not a prerequisite to calling `ConvertSpaceToModel`.